base_app/package.json

```diff
@@ -16,10 +16,11 @@
     "start-client-dev": "cd ../ && bash run-dev-env.bash",
     "start-client-prod": "bash -c \"node node_modules/@angular/cli/bin/ng.js serve --aot\"",
     "post-build-step--setup-bms-for-dev": "sh build-app-for-dev.sh 0 bms git-workspace-all ${USER}",
-    "test": "node node_modules/@angular/cli/bin/ng.js test --browsers Chrome_small",
+    "test": "node node_modules/@angular/cli/bin/ng.js test --browsers ChromeHeadless",
```

Comment:
Originally the `test` target using `Chrome_small` was intended to allow a manual sanity test in a visible small window in which UI transitions could be observed. I agree with making the headless version the default, but I've seen you demonstrate automated tests with an active browser window.  Is there any value in having a `test-visible-ui` target?  It's ok this is ok as is.

Reply:
hmm good question. Im not sure if there is a good reason for a headful target. Since we are just doing unit tests, I dont see much benefit but Im open to ideas. I think E2E tests should definitely be headful though

Reply:
ok - leave as is (headless).